sign/rpmgensig.c

```diff
@@ -396,6 +397,16 @@ static void deleteSigs(Header sigh)
     headerDel(sigh, RPMSIGTAG_PGP5);
 }
 
+static void deleteFileSigs(Header sigh)
+{
+    headerDel(sigh, RPMSIGTAG_FILESIGNATURELENGTH);
+    headerDel(sigh, RPMSIGTAG_FILESIGNATURES);
+    headerDel(sigh, RPMTAG_VERITYSIGNATURELENGTH);
+    headerDel(sigh, RPMTAG_VERITYSIGNATURES);
+    headerDel(sigh, RPMTAG_VERITYSIGNATUREALGO);
+    headerDel(sigh, RPMTAG_VERITYSIGNATUREBLKSZ);
```

Comment:
This deals with a signature header, so RPMSIGTAG_* values should be used (even if they actually match the RPMTAG_* counterparts)

Reply:
Thanks, I actually discovered this the hard way and already fixed it in my tree.